edx/edx-exams
# feat: update latest attempt endpoint

**JIRA:** [MST-1859](https://2u-internal.atlassian.net/browse/MST-1859)

**Description:** This PR updates the get latest endpoint to return latest regardless of status and to fix logic so that edx-proctoring call return of no active attempts doesn't return logic to return regardless of status.

**Testing instructions:**

1. [Set up LTI](https://2u-internal.atlassian.net/wiki/spaces/PT/pages/256737327/Local+Development+and+LTI+Configuration)
2. Open proctored exam as learner
3. Click through interstitials and click on Start System Check
4. Once LTI check passes, click Start Exam and should go to next ready to start interstitial
5. If reload is required before showing next interstitial, then failed check

**Merge checklist:**

**Post merge:**
